pages/profile-edit/form/fields.js

```diff
@@ -33,58 +30,53 @@ module.exports = function(showLongBio) {
     },
     user_bio: {
       type: `textarea`,
-      label: (
-        <div>
-          Bio <em>(do not include any personal information)</em>
-        </div>
-      ),
+      label: `Bio`,
       placeholder: `About you`,
+      labelClassname: `h5-heading bio`,
       fieldClassname: `form-control`,
       validator: [validator.maxLengthValidator(140)],
       charLimit: 140,
       charLimitText: function(charCount, charLimit) {
         return charLimit - charCount;
       }
     },
-    twitter: {
+    Twitter: {
```

Comment:
Why are we changing this? This changes the input field's `name` attribute value from `twitter` to `Twitter`.

Reply:
It also corresponds to the element's class and in this case, adds a glyph before each `label`.